wolox-books/src/environments/environment.ts

```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  base_url: 'https://wbooks-api-stage.herokuapp.com/api/v1'
```

Comment:
the environment variables should not be pushed to github.. add this file to the gitignore